### PR TITLE
Features/input image

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
**Image and Input Component**
1. Square image
2. Circular image
3. Full width responsive image
4. Label and input field for user name and password

### CODE REVIEW

**Review by sanjay-xdr — APPROVED** [review]

File: docs.html [hunk 1]
```diff
@@ -794,6 +800,49 @@ <h3>Horizontal Product Card landscape</h3>
             ></iframe>
           </div>
         </div>
+
+        <div id="image">
```
Comment (amirnathani9) [inline docs.html:798]: do proper naming convention

**Comment by amirnathani9** [comment]
Looks Good to ME!!! Only one Suggestion to change.

File: docs.html (hunk 1 above)
Comment (Rishi05051997) [reply docs.html:798]: Looking Good Now

File: Components/avatar.css [hunk 2]
```diff
@@ -31,16 +31,17 @@
...
-.circular-avatar {
+.circular-img {
   border-radius: 50%;
 }
 .relative-circular-avatar {
   border-radius: 50%;
   position: relative;
 }
-.img {
+.responsive-img {
```
Comment (Rishi05051997) [inline Components/avatar.css:41]: used max-inline-size:100%, block-size: auto for image responsiveness

**Review by Rishi05051997 — COMMENTED** [review]
Look cool please do suggested changes

**Pull request merged**